tests/src/Functional/AlertBannerBlockTest.php

```diff
@@ -76,4 +76,69 @@ public function testNonLiveAlertBannerDoesNotDisplay() {
     $this->assertSession()->pageTextNotContains($alert_message);
   }
 
+  /**
+   * Test display title option.
+   */
+  public function testAlertDisplayTitle() {
+    $title = $this->randomMachineName(8);
+    $alert_message = 'Alert message: ' . $this->randomMachineName(16);
+    $alert = $this->container->get('entity_type.manager')->getStorage('localgov_alert_banner')
+      ->create([
+        'type' => 'localgov_alert_banner',
+        'title' => $title,
+        'short_description' => $alert_message,
+        'type_of_alert' => 'minor',
+        'display_title' => 1
+      ]);
+    $alert->save();
+
+    // Flag the alert banner to put it live.
+    $flag_service = $this->container->get('flag');
+    $flag = $flag_service->getFlagById('localgov_put_live');
+    $flag_service->flag($flag, $alert);
+
+    // Check title is shown when display title is true
+    $this->drupalGet('<front>');
+    $this->assertSession()->pageTextContains($title);
+
+    // Check title is not shown when display title is false
+    $alert->set('display_title', ['value' => 0]);
+    $alert->save();
+    $this->drupalGet('<front>');
+    $this->assertSession()->pageTextNotContains($title);
+  }
+
+  /**
+   * Test remove hide link option.
+   */
+  public function testAlertRemoveHideLink() {
+    $title = $this->randomMachineName(8);
+    $alert_message = 'Alert message: ' . $this->randomMachineName(16);
+    $alert = $this->container->get('entity_type.manager')->getStorage('localgov_alert_banner')
+      ->create([
+        'type' => 'localgov_alert_banner',
+        'title' => $title,
+        'short_description' => $alert_message,
+        'type_of_alert' => 'minor',
+        'remove_hide_link' => 0
+      ]);
+    $alert->save();
+
+    // Flag the alert banner to put it live.
+    $flag_service = $this->container->get('flag');
+    $flag = $flag_service->getFlagById('localgov_put_live');
+    $flag_service->flag($flag, $alert);
+
+    // Check hide link is shown when remove_hide_link is not set
+    $this->drupalGet('<front>');
+    $this->assertSession()->responseContains('js-alert-banner-close');
+    $this->assertSession()->pageTextContains('Hide');
```

Comment:
Not sure we need this as we already test for `->responseContains('js-alert-banner-close');`